src/core/core.scale.js

```diff
@@ -523,6 +523,10 @@ module.exports = Element.extend({
 		if ((typeof rawValue === 'number' || rawValue instanceof Number) && !isFinite(rawValue)) {
 			return NaN;
 		}
+		// Float-bar support. Handling arrays
+		if (helpers.isArray(rawValue)) {
+			return [+this.getRightValue(rawValue[0]), +this.getRightValue(rawValue[1])];
```

Comment:
Aren't we going to break custom implementation that expose their data as array with more than 2 items?

May be safer to iterate the whole array:
```
return rawValue.map(function(v) {
    return this.getRightValue(v);
});
```

Also, why do we need to convert to number (`+`)? For example, it could be time strings or Date() or ...

Reply:
this conversion was before me. I have followed the same way to prevent tests failing cause some tests expected it to be integer. So, can we keep it?

Reply:
About array iteration, i am not sure if we need to iterate whole array. This functionality expects only 2 values in array and it used only for float bars. So, of course we can add iteration but at the moment we never expect more than 2 values in array.

Reply:
@gwyneblaidd previously the `+` was in `getLabelForIndex` and you removed it there, which is why it's now required here. Perhaps you could remove it here and put it back in `getLabelForIndex`?

Reply:
> So, can we keep it?

I can't remember where this comes from but it doesn't make sense to have this conversion here. And for consistency, none of the other paths return the value converted as number so I would remove `+`.

> This functionality expects only 2 ...

This method doesn't know how the result will be handled so I would apply the same logic to all items. But what's the reason to `getRightValue` on array item? Could we simply return `rawValue` if it's an array (i.e. not handle array here)?

Reply:
I suppose I would add the array iteration since it doesn't hurt and is more compatible with custom plugins that users may choose to write taking more than two values as @simonbrunel suggested

Reply:
Just returning `rawValue` here makes sense to me after `+` is moved back to `getLabelForIndex`

Reply:
`getLabelForIndex` returns `getScaleLabel` which already handles `+` (and `getScaleLabel` can return a string, so would break it).

Reply:
done